lib/omniauth/strategies/oidc/back_channel_logout_request.rb

```diff
@@ -0,0 +1,81 @@
+module OmniAuth
+  module Strategies
+    class Authentiq
+      class BackChannelLogoutRequest
+
+        def initialize(strategy, request)
+          @strategy, @request = strategy, request
+        end
+
+        def call(options = {})
+          @options = options
+
+          begin
+            # for
+            # sign_out_callback.call(*back_channel_logout_request)
+            # to actually happen
+            # a proc must be set in the devise.rb initializer of the rails app
+            result = sign_out_callback.call(*back_channel_logout_request)
+          rescue
+            return backchannel_logout_response(400, ['Bad request']).finish
```

Comment:
Hm I though we should go according to this

```
2.8.  Back-Channel Logout Response

If the logout succeeded, the RP MUST respond with HTTP 200 OK. If the logout request was invalid, the RP MUST respond with HTTP 400 Bad Request. If the logout failed, the RP MUST respond with 501 Not Implemented. If the local logout succeeded but some downstream logouts have failed, the RP MUST respond with HTTP 504 Gateway Timeout.

The RP's response SHOULD include Cache-Control directives keeping the response from being cached to prevent cached responses from interfering with future logout requests. It is RECOMMENDED that these directives be used:

  Cache-Control: no-cache, no-store
  Pragma: no-cache

```